packages/accounts/src/msca/plugins/multi-owner/extension.ts

```diff
@@ -19,11 +19,25 @@ export type MultiOwnerPluginActions<
   readOwners: (
     params: { pluginAddress?: Address } & GetAccountParameter<TAccount>
   ) => Promise<ReadonlyArray<Address>>;
+
+  isOwnerOf: (
+    address: Address,
+    params?: {
+      pluginAddress?: Address;
+    } & GetAccountParameter<TAccount>
+  ) => Promise<boolean>;
```

Comment:
This was right before
```suggestion
  isOwnerOf: (
    params: {
      address: Address,
      pluginAddress?: Address;
    } & GetAccountParameter<TAccount>
  ) => Promise<boolean>;
```

Reply:
we should stay consistent with the style of all our other calls. so they should all take in an object as params